File: FullstackDeveloperAssessment.Domain/Entyties/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;
using FullstackDeveloperAssessment.Domain.Enumerators;

namespace FullstackDeveloperAssessment.Domain.Entyties
{
	public class Person
	{
				
		public int VAT { get; set; }
		public string Name { get; set; }		
		public PersonAddress PersonAddress { get; set; }
		public DateTime DateOfBirth { get; set; }
		public IEnumerable<PersonPersonType> PersonPersonTypes { get; set; }		
		public Gender Gender { get; set; }
		public string SecretCode { get; set; }
		public DateTime DateOfRecord { get; set; }

	}



}

File: poc-frontend/src/models/person.ts
import { Address } from './address';
import { PersonPersonType } from './personPersonType';


export class Person {
    public Vat: number;
    public Name: string;
    public PersonAddress: Address;
    public DateOfBirth: Date;
    public PersonPersonTypes: PersonPersonType[];
    public Gender: Gender;
    public SecretCode: string;
    public DateOfRecord: Date;
}

enum Gender {
    Male = 1,
    Female = 2
}